README.md

```diff
@@ -138,63 +131,163 @@ In support of the various browser privacy models, first-party sets only control
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>ccTLD (country code Top Level Domain)</td>
+      <td>Reserved for variations for a particular country or a geographical area. <br>
+<br>
+Requires common ownership.</td>
+      <td>No limit on domains, auto-grant access</td>
+    </tr>
+    <tr>
+      <td>common eTLD (effective Top Level Domain)</td>
+      <td>Reserved for domains that share a common eTLD as the set primary. These are not <a href="https://www.iana.org/domains/root/db">IANA-managed TLDs</a>, but domains added to the <a href="https://publicsuffix.org/">PSL</a> for improved security isolation.<br>
+<br>
+Requires common ownership.</td>
+      <td>No limit on domains, auto-grant access</td>
+    </tr>
+    <tr>
+      <td>service</td>
+      <td>Reserved for utility or sandbox domains.<br>
+<br>
+Requires common ownership.</td>
+      <td>No limit on domains, auto-grant access. Not allowed to be the top-level domain in a storage access grant.</td>
+    </tr>
+    <tr>
+      <td>associated</td>
+      <td>Reserved for domains whose affiliation with the set primary is clearly presented to users (e.g., an About page, header or footer, shared branding or logo, or similar forms).</td>
+      <td>Limit of 3* domains. If greater than 3, auto-reject access.<br>
+<br>
+<em>*[^1]exact number TBD</em></a></td>
+    </tr>
+  </tbody>
+</table>
+
+### Abuse mitigation measures
+
+We consider using a public submission process (like a GitHub repository) to be a valuable approach because it facilitates our goal to keep all set submissions public and submitters accountable to users, civil society, and the broader web ecosystem. For example, a mechanism to report potentially invalid sets may be provisioned. We expect public accountability to be a significant deterrent for intentionally miscategorized subsets.
+
+The following technical checks also help to mitigate abuse:
+
+-   Mutual exclusivity to ensure a domain isn't part of multiple First-Party Sets
+-   `.well-known` file check on all domains to ensure authorized submissions
+-   Check against the [Public Suffix List](https://publicsuffix.org/) to ensure that sets are composed of valid registrable domains
+
+Additionally, there are other enforcement strategies we could consider to further mitigate abuse. If there is a report regarding a domain specified under the "service" subset, potential reactive enforcement measures could be taken to validate that the domain in question is indeed a "service" subset.
+
+For some subsets, like the "associated" subset, objective enforcement may be much more difficult and complex. In these situations, the browser's handling policy, such as a limit of three domains, should limit the scope of potential abuse. Additionally, we think that site authors will be beholden to the subset definition and avoid intentional miscategorization as their submissions would be entirely public and constitute an assertion
```

Comment:
One likely risk here is that the repository gets filled with long discussions of people's subjective evaluation of whether a common context or branding is "clearly presented." Limiting the number of domains would reduce the payoff for winning one argument about whether a set is clear to the user, but could increase the number of such arguments.

Graphic design conventions, and of course text about a site's set membership, can also be different across languages and cultures. A rule based on posting user research to the repository (see my comment on line 252) could help to limit the number and length of arguments about "clear" sets.

Reply:
As mentioned, if the primary idea for user research is to discourage discussion, maybe it makes more sense specifically for organizations that expect such discussion to occur to prepare such research, and not for smaller sites or sites that are confident that their use case is clear enough that it won't come under public scrutiny.